lib/formdata/constants.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const states = {
+  INITIAL: 0,
+  BOUNDARY: 1,
+  READ_HEADERS: 2,
+  READ_BODY: 3
+}
+
+const headerStates = {
+  DEFAULT: -1, // no match
+  FIRST: 0,
+  SECOND: 1,
+  THIRD: 2
+}
+
+const chars = {
+  '-': '-'.charCodeAt(0),
+  cr: '\r'.charCodeAt(0),
+  lf: '\n'.charCodeAt(0),
+  ':': ':'.charCodeAt(0),
+  ' ': ' '.charCodeAt(0), // 0x20
+  ';': ';'.charCodeAt(0),
+  '=': '='.charCodeAt(0),
+  '"': '"'.charCodeAt(0)
+}
+
+const emptyBuffer = Buffer.alloc(0)
+
+const crlfBuffer = Buffer.from('\r\n')
```

Comment:
Nit: Can we pre calculate this and write it here to reduce the boot time overhead, even though it might be negligible?